src/components/VideoCard/VideoCard.component.jsx

```diff
@@ -8,11 +9,27 @@ import {
 } from './VideoCard.styled';
 
 const VideoCard = (props) => {
+  const currentVideoContext = React.useContext(CurrentVideoContext);
+  const onClickHandler = () => {
+    currentVideoContext.setVideoDetails({
+      id: props.id,
```

Comment:
it's a good practice to destructure the props component and use them the single word
`const { id = '', title = '', description = ''} = props`

or destructure them in the params
`const VideCard = ({ id = '', title = '', description = ''} ) => {`

Reply:
Went with the first option!